a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -61,7 +61,7 @@ def execute(self, amphora):
         return vthunder
 
 
-class CreteVthunderEntry(BaseDatabaseTask):
+class CreateVthunderEntry(BaseDatabaseTask):
```

Comment:
We have resolved this in last PR, you might get a conflict here

Reply:
That's fine. Once we merge those PRs I'll rebase this against master.